tareas/clase-6/tarea-1/calculos-para-la-tarea-1.js

```diff
@@ -0,0 +1,27 @@
+function obtenerEdadMayorEdad(edadesIntegrantes) {
+    let mayorEdad = edadesIntegrantes[0];
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        if (edadesIntegrantes[i] > mayorEdad) {
+            mayorEdad = edadesIntegrantes[i];
+        }
+    }
+    return mayorEdad;
+}
+
+function obtenerEdadMenorEdad(edadesIntegrantes) {
+    let edadMinima = edadesIntegrantes[0];
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        if (edadesIntegrantes[i] < edadMinima) {
+            edadMinima = edadesIntegrantes[i];
+        }
+    }
+    return edadMinima;
+}
+
+function calcularPromedio(edadesIntegrantes) {
+    let resultado = 0;
+    for (let i = 0; i < edadesIntegrantes.length; i++) {
+        resultado = resultado + edadesIntegrantes[i];
+    }
+    return (promedio = resultado / edadesIntegrantes.length);
```

Comment:
No te hacen falta los paréntesis y no te hace falta declarar `promedio` porque nunca la usás
```suggestion
    return resultado / edadesIntegrantes.length;
```